### ISSUE TITLE
[feature require] pointer->udev-device and udev->pointer.

### ISSUE THREAD

**Issue by Z572** [artyom-poptsov/guile-udev#6]
libinput have `libinput_udev_create_context` and `libinput_device_get_udev_device` require udev and udev_device struct

If guile-udev provide `udev->pointer` and `pointer->udev-device`, i can use them at [guile-libinput](https://github.com/Z572/guile-libinput)

**Comment by artyom-poptsov** [comment]
Hello,

please check this branch:
https://github.com/artyom-poptsov/guile-udev/tree/wip-udev-pointers

Does it look like something that you requested?

This is new for me so I want to make sure I understood the feature request correctly.

Thanks!

-- avp

**Comment by Z572** [comment]
Yes this is what I need, but there are problems with this version, for example `pointer->udev` executing the same pointer twice should return the same udev, see `define-wrapped-pointer-type`. It uses `make-weak-value-hash-table` internally
